style.css

```diff
@@ -86,3 +95,160 @@ main {
 .arrow-icon {
   text-align: center;
 }
+
+.flex {
+  display: flex;
+  padding-bottom: 20px;
+  width: 100%;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.works-section-title {
+  text-align: center;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.works-grid-container {
+  display: grid;
+  gap: 24px;
+  justify-content: center;
+  width: 86%;
+  margin: 62px auto;
+}
+
+.card img {
+  width: 327px;
+  max-width: 100%;
+  height: 252px;
+}
+
+.card {
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+button {
+  cursor: pointer;
+}
+
+.card-details {
+  align-items: center;
+  gap: 12px;
+  color: #172b4d;
+}
+
+.card-details h2 {
+  margin-top: 16px;
+}
+
+.card-details ul {
+  margin-right: 20px;
+}
+
+.list {
+  gap: 8px;
+}
+
+.card-details button {
+  margin: 15px 102px 24px 103px;
+  padding: 12px;
+  background-color: #36b37f;
+  border: none;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.about-me {
+  background-image: url(./Icons/about-top.svg), url(./Icons/about_bg.svg);
+  background-repeat: no-repeat;
+  background-position: right -40px, 0 270px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design:
- ![image](https://user-images.githubusercontent.com/60083437/227739680-b2b7ba2c-a34f-4f95-929e-d6c9249d378b.png)

Reply:
- As suggested by the previous reviewer, please, kindly use the correct fonts as used in the Figma design to improve your design.